src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String REGEX_CONST = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
this method looks a bit big, let's move writing to file and forming result into separate private methods